### PR TITLE
Feature/add sales invoices endpoint

### PR DESCRIPTION
**Pull request by Naoray**

### CODE REVIEW

File: examples/sales-invoices/create-sales-invoice.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    /*
+     * Create a sales invoice
+     */
+    $salesInvoice = $mollie->salesInvoices->create([
+        'currency' => 'EUR',
+        'status' => SalesInvoiceStatus::DRAFT,
+        'vatScheme' => 'standard',
+        'vatMode' => 'inclusive',
+        'paymentTerm' => PaymentTerm::DAYS_30,
```
Comment (sandervanhooft) [inline examples/sales-invoices/create-sales-invoice.php:24]: This Type does not exist, perhaps for the better.

File: phpstan-baseline.neon [hunk 2]
```diff
@@ -220,6 +220,31 @@ parameters:
 			count: 1
 			path: examples/profiles/update-profile.php
 
+		-
+			message: "#^Access to constant DAYS_30 on an unknown class Mollie\\\\Api\\\\Types\\\\PaymentTerm\\.$#"
```
Comment (sandervanhooft) [inline phpstan-baseline.neon:224]: Phpstan picked up on it too

**Review by sandervanhooft — CHANGES_REQUESTED** [review]

File: examples/sales-invoices/create-sales-invoice.php (hunk 1 above)
Comment (Naoray) [reply examples/sales-invoices/create-sales-invoice.php]: I don't think having this type d be bad.

Comment (sandervanhooft) [reply examples/sales-invoices/create-sales-invoice.php]: Let's add it :+1:

File: src/Types/SalesInvoiceStatus.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Mollie\Api\Types;
+
+class SalesInvoiceStatus
+{
+    /**
+     * The sales invoice is in draft status and has not been sent or paid.
+     */
+    const DRAFT = 'draft';
```
Comment (sandervanhooft) [inline src/Types/SalesInvoiceStatus.php:10]: These constants can be marked as `public`

**Review by sandervanhooft — APPROVED** [review]

**Pull request merged**